modules/ytp-assets-common/src/less/ckan/upstream_ckan/module.less

```diff
@@ -175,3 +193,12 @@
         vertical-align: middle;
     }
 }
+
+.apiset-title {
+  color: rgb(34, 34, 34);
+  font-size: 40px;
+  font-weight: 300;
+  height: 48px;
+  line-height: 48px;
+  margin: 0 0 3px;
+}
```

Comment:
Less files under `upstream_ckan/` shouldn't be touched, so these should be moved into some other place